environment.yml

```diff
@@ -15,4 +15,5 @@ dependencies:
 - bokeh
 - markdown
 - pip:
-  - compdevkit
\ No newline at end of file
+  - cs-kit
+  - s3like
```

Comment:
@Peter-Metz you shouldn't need s3like anymore. Are you getting import errors from not having s3like installed when running the tests?

Reply:
@hdoupe yeah. The Travis build was failing until I added s3like:

```python
_________ ERROR collecting cs-config/cs_config/tests/test_functions.py _________
ImportError while importing test module '/home/travis/build/PSLmodels/Tax-Cruncher/cs-config/cs_config/tests/test_functions.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
cs-config/cs_config/tests/test_functions.py:1: in <module>
    from cs_kit import CoreTestFunctions
../../../miniconda/envs/taxcrunch-env/lib/python3.7/site-packages/cs_kit/__init__.py:3: in <module>
    from .validate import CoreTestFunctions
../../../miniconda/envs/taxcrunch-env/lib/python3.7/site-packages/cs_kit/validate.py:10: in <module>
    import s3like
E   ModuleNotFoundError: No module named 's3like'

Reply:
Want to try it now? I just pushed 1.8.1 which removes the use of `s3like`.

Reply:
Yep working now

Reply:
Sweeetttt.